quickevent/app/plugins/qml/Runs/src/runswidget.h

```diff
@@ -62,13 +62,18 @@ private slots:
 	QList< QList<int> > runnersByClubSortedByCount(int stage_id, int class_id, QMap<int, QString> &runner_id_to_club);
 	QList<int> runsForClass(int stage_id, int class_id);
```

Comment:
It looks to me like this comment refers to an outdated commit. With introduction of the API enhancement, code changes in runswidget.h are limited to new values in DrawMethod enum. The patch set no longer touches function prototypes. CMIIW